### ISSUE TITLE
The word "constructor" cannot be used as a key or table name

### ISSUE THREAD

**Issue by atdyer** [BinaryMuse/toml-node#52]
To recreate, open the [live demo](http://binarymuse.github.io/toml-node/) and replace any key or table name with the word `constructor`. Even using quotes around the word does not prevent crashing.

Creating an empty table with the name constructor does not cause a crash, but simply omits the empty table altogether:

```toml
[empty]
[emptier]
[constructor]
[emptiest]
```
results in
```js
{
  "empty": {},
  "emptier": {},
  "emptiest": {}
}
```

**Comment by BinaryMuse** [comment]
This is fixed in v2.3.4

**Comment by BinaryMuse** [comment]
I lied! 🙈 This is fully fixed in v2.3.5.

**Comment by atdyer** [comment]
Excellent! Thanks for such a quick turnaround :+1:

**Comment by BinaryMuse** [comment]
Due to https://github.com/BinaryMuse/toml-node/issues/55, this has been reverted in `v2.3.6` but is present in the newly published `v3.0.0`.
